pandaserver/dataservice/closer.py

```diff
@@ -0,0 +1,469 @@
+"""
+update dataset DB, and then close dataset and start Activator if needed
+
+"""
+
+import re
+import sys
+import datetime
+
+from typing import Dict, List
+
+from pandacommon.pandalogger.PandaLogger import PandaLogger
+from pandacommon.pandalogger.LogWrapper import LogWrapper
+
+from pandaserver.config import panda_config
+from pandaserver.dataservice import Notifier
+from pandaserver.dataservice.Activator import Activator
+from pandaserver.taskbuffer import EventServiceUtils
+
+# logger
+_logger = PandaLogger().getLogger("closer")
+
+
+def init_logger(p_logger: PandaLogger) -> None:
```

Comment:
remove this and decommission notifier.py

Reply:
hm EventPicker is using Notifier in the sendEmail function, should I comment this out from there as well?

Reply:
Yes, the notification is historical and rather useless.